src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -165,6 +166,19 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     event_mask |= COR_PRF_DISABLE_OPTIMIZATIONS;
   }
 
+  const WSTRING domain_neutral_instrumentation =
+      GetEnvironmentValue(environment::domain_neutral_instrumentation);
+
+  if (domain_neutral_instrumentation == "1"_W || domain_neutral_instrumentation == "true"_W) {
+    Info("Detected environment variable ", environment::domain_neutral_instrumentation,
+         "=", domain_neutral_instrumentation);
+    Info("Enabling automatic instrumentation of methods called from domain-neutral assemblies. ",
+         "Please ensure that there is only one AppDomain or, if applications are being hosted in IIS, ",
+         "ensure that all Application Pools only have one application. ",
```

Comment:
Suggest adding `each`, so it becomes `ensure that all Application Pools only have one application each.`

Reply:
I would also say "at most one" since Application Pools can have zero applications.
```suggestion
         "ensure that all Application Pools have at most one application each. ",
```

Reply:
I applied both `each` and `at most one` language in 1a79b7b5239ab20af134e26484cca6db56e29cb5